backend/node/genvm/calldata.py

```diff
@@ -0,0 +1,198 @@
+from .types import Address
```

Comment:
I'm assuming this file is copied/adapted from the GenVM codebase right? It's very cryptic to me

Reply:
Yes, it is
Unfortunately genvm repo is still private, even though I asked JM about making it public. Docs for calldata are [there](https://github.com/yeagerai/genvm/blob/main/doc/calldata.md)